README.md

```diff
@@ -1 +1,38 @@
 # Pipeline Run Prometheus Counter
+
+Expose counter metrics for various pipeline runs for the Prometheus text format.
+
+This is useful if you have a pipeline such as Tekton or Jenkins and you need
+to keep track of success and/or failure rates of Pipeline runs in Prometheus.
+
+These pipeline runs are stored in a simple SQLite database.
+
+## HTTP Endpoints
+
+You may view the Prometheus metrics at the `/metrics` endpoint. The ouput
+will be in the following format:
+
+```text
+# HELP car_factory_pipeline_run_total The total number of pipeline runs.
+# TYPE pipeline_tests_pipeline_run_total counter
+car_factory_pipeline_run_total{status="success"} 1
+car_factory_pipeline_run_total{status="failed"} 1
```

Comment:
There are separate pipelines (e.g. dev/stage/prod) and additionally pipeline runs can be triggered with specific parameters. Would it make sense to have this categorized by pipeline run prefix? Or is the `car_factory` prefix used for this? In such case, I expect there would need to be multiple pods collecting metrics, but this would consume more resources than necessary.

Reply:
@hluk if you think we want to store the metrics for those pipelines too, I'd be happy make this configurable.

Reply:
That would be useful. If it would be used just for prod for our greenwave segment tests, then we two separate prod pipelines - one for old resultsdb-updater and other for the new result message consumer.

Reply:
@hluk I now made it so that a name must be provided in the POST endpoint. We can now use this for multiple pipelines.